data/tools/Android.json

```diff
@@ -0,0 +1,40 @@
+{
+  "tools": [
+  
+      {
+      "name": "Andriller",
+      "source": "https://github.com/den4uk/andriller",
+      "description": "Software utility with a collection of forensic tools for smartphones.",
+      "language": "Python",
+      "price": "Free",
+      "online": "False"
+    },
+    
+    {
+      "name": "Androrat",
+      "source": "https://github.com/wszf/androrat",
```

Comment:
I don't know if a RAT would be put rather in the network or red team category, but since there isn't any really legal usage of a RAT that are used only for illegal purposes I would suggest not to add it to the inventory.